### PR TITLE
Updated instructions

### PR DESCRIPTION
**Pull request by jbnunn**
Added `--save` to sample code. Removed extra `cd lambda/custom`

*Issue #, if available:*

*Description of changes:*
Made instructions more clear to those who are following along and copy/pasting.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: instructions/2-lambda-function.md [hunk 1]
```diff
@@ -14,13 +14,12 @@
 
 	```bash
 	$ cd lambda/custom
-	$ npm install
+	$ npm install --save
```
Comment (mdinescu) [inline instructions/2-lambda-function.md]: the `save` option is not needed: did you find that you actually needed it?

It looks like the text above should be updated to remove the `--save` parameter from there.

**Review by mdinescu — CHANGES_REQUESTED** [review]

File: instructions/2-lambda-function.md (hunk 1 above)
Comment (jbnunn) [reply instructions/2-lambda-function.md]: I didn't need it, you're right. We might want to update line 13 then to make the instructions consistent with the code sample.

Comment (mdinescu) [reply instructions/2-lambda-function.md]: Can you update the pull request? Or would you prefer that we close it and I make the changes?

**Review by jbnunn — COMMENTED** [review]
Updated -- thank you

**Review by mdinescu — APPROVED** [review]
Looks good! Thanks for taking the time to update the instructions

**Comment by jbnunn** [comment]
Hi @Theyseemeroman, see [Step 3](https://github.com/alexa/skill-sample-nodejs-buttons-trivia/blob/master/instructions/2-lambda-function.md) in Prepare Your Code for Deployment. You'll create a ZIP archive via the command line in your terminal, then upload it as shown in Step 13.

**Pull request merged**